website/project/model.py

```diff
@@ -2158,8 +2159,13 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             foreign_user=foreign_user,
             params=params,
         )
+
         if log_date:
             log.date = log_date
+
+        self.date_modified = log.date.replace(tzinfo=None)
```

Comment:
Consulted with @brianjgeiger to confirm that it makes sense to not include it in the dict. The `date_modified` field is implicitly updated, so, like `logs`, it doesn't need to be serialized.